components/NavBar.js

```diff
@@ -268,20 +284,32 @@ const NavBar = () => {
   return (
     <NavBarContainer visibility={visibility} opacity={opacity}>
       <NavGroupContainer>
-        <NwPlusLogo src="/assets/logos/nwPlus_Logo_2020.svg" alt="nwPlus club logo in white against dark blue background"/>
+        <a href='https://nwplus.io'>
```

Comment:
link to homepage

Reply:
Same comment as above - can change to `/`